src/util/permissions.ts

```diff
@@ -0,0 +1,31 @@
+import {IObservableArray} from 'mobx';
+import {PermissionInfo} from '@cuba-platform/rest';
+
+export enum PermissionType {
+  // noinspection JSUnusedGlobalSymbols
+  SCREEN = "SCREEN",
+  ENTITY_OP = "ENTITY_OP",
+  ENTITY_ATTR = "ENTITY_ATTR",
+  SPECIFIC = "SPECIFIC",
+  UI = "UI"
+}
+
+export enum PermissionValue {
+  // noinspection JSUnusedGlobalSymbols
+  ALLOW = "ALLOW",
+  DENY = "DENY"
+}
+
+export function isFieldAllowed(entity: string, field: string, perms: IObservableArray<PermissionInfo> | undefined): boolean {
```

Comment:
I suggest more generic name like `isAttributeAllowed`

Reply:
Also I think we need to provide more clear and separate functions for each operation: read/update